### PR TITLE
Jw nonprofit dashboard

### PR DESCRIPTION
**Pull request by jenkens-dev**

### CODE REVIEW

**Comment by jenkens-dev** [comment]
@Alphy11

File: src/components/GifButton.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import React from 'react';
+
+// Functional component that displays a clickable gif with a caption
+// Layout:
+// Image, Caption
+
+type Props = {
+  caption: string;
+  image: string;
+  containerStyle?: object;
```
Comment (Alphy11) [inline src/components/GifButton.tsx]: would just call this style, or even better would take in a `classname` prop, and append to classname on line 17

File: src/components/NonprofitGraph.tsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const NonprofitGraph = (props: { displayOption: String }) => {
```
Comment (Alphy11) [inline src/components/NonprofitGraph.tsx:3]: This is a placeholder I assume?

File: src/components/NonprofitGraphDisplay.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import NonprofitGraph from './NonprofitGraph';
+
+const NonprofitGraphDisplay = () => {
+  const [displayOption, setDisplayOption] = useState('metrics');
```
Comment (Alphy11) [inline src/components/NonprofitGraphDisplay.tsx]: you should type your useState calls.

```ts
type DisplayOption = 'metrics' | 'events' | 'volunteers';
/* ... */
  const [displayOption, setDisplayOption] = useState<DisplayOption>('metrics');


```

**Review by Alphy11 — COMMENTED** [review]
@Taljjaa took a look

File: src/components/NonprofitGraph.tsx (hunk 2 above)
Comment (jenkens-dev) [reply src/components/NonprofitGraph.tsx:3]: Yes I need to learn D3

**Pull request merged**